darts/models/rnn_model.py

```diff
@@ -145,12 +149,22 @@ def __init__(self,
             Sizes of hidden layers connecting the last hidden layer of the RNN module to the output, if any.
         dropout
             Fraction of neurons afected by Dropout.
+        random_state
+            Control the randomness of the weights initialization. Check this
+            `link <https://scikit-learn.org/stable/glossary.html#term-random-state>`_ for more details.
         """
 
         kwargs['output_length'] = output_length
         kwargs['input_size'] = input_size
         kwargs['output_size'] = output_size
 
+        # TODO : make it a util function? -> reusable in other torch models that needs fixed seed...
+        # set the random seed
```

Comment:
I think it should be a util function used in every torch model that need fixed seed. What do you think ?

Reply:
Agreed, I think it could be one of kwargs in ForecastingModel and set there if possible - just need to make sure fixing the seed in one class will not affect all of the other ones.

Reply:
Perhaps it is enough to set it in `TorchForecastingModel` (at least for now)